### DISCUSSION TITLE
About Security Course

### DISCUSSION THREAD

**Discussion by MostPow3rful** [Cyfrin/foundry-full-course-cu#2787] [General]
GM GM
i know basic of blockchain tech, solidity programming language, i have done the foundry101 course and my question is: can i start the Updraft's Security course or not? ( i have some basic knowledge of vulnerabilities and exploited them )

**Comment by EngrPips** [comment]
why haven't you done the `Advance` Foundry course?

**Reply by MostPow3rful** [reply]
i started it but i stuck at `stablecoin` and it's so hard and i want to know that can i jump to security course or not ?

**Comment by EngrPips** [comment]
No you can't. You need to finish it.

**Reply by MostPow3rful** [reply]
Really? .... any advice ? beacuse it's really hard to learn and develop 0_0

**Reply by BowTiedOffSec** [reply]
This lays the groundwork to become a security researcher and is an important step, you need to make sure you have a solid understanding of how solidity functions.  That being said, if you have already found exploits, have you considered competing in audits and such?  Then you will find your current weak points and start to get a feel for what you should work on.

**Reply by EngrPips** [reply]
Well said @BowTiedOffSec
